integration-cli/docker_cli_network_unix_test.go

```diff
@@ -1746,13 +1746,20 @@ func (s *DockerNetworkSuite) TestConntrackFlowsLeak(c *testing.T) {
 	cli.DockerCmd(c, "network", "create", "--subnet=192.168.10.0/24", "--gateway=192.168.10.1", "-o", "com.docker.network.bridge.host_binding_ipv4=192.168.10.1", "testbind")
 	assertNwIsAvailable(c, "testbind")
 
+	// Replaces the image built from https://github.com/appropriate/docker-nc/tree/master/latest
+	imgName := "moby/nc"
+	icmd.RunCmd(icmd.Cmd{
+		Command: []string{dockerBinary, "build", "-t", imgName, "-"},
+		Stdin:   strings.NewReader("FROM alpine:3.14\nRUN apk add --no-cache netcat-openbsd\n"),
+	}).Assert(c, icmd.Success)
+
 	// Launch the server, this will remain listening on an exposed port and reply to any request in a ping/pong fashion
 	cmd := "while true; do echo hello | nc -w 1 -lu 8080; done"
```

Comment:
If we switch to the (already in our "frozen images") `busybox` image, I've had success with the following returning data correctly:

```suggestion
	cmd := "while true; do echo hello | nc -w 1 -l -u -p 8080; done"
```